src/new-client/src/app/lib/components/site-list.test.tsx

```diff
@@ -13,6 +13,23 @@ describe('<SiteList>', () => {
     expect(screen.queryByRole('link', { name: 'Site Beta' })).toBeVisible();
   });
 
+  it('renders sites in ascending alphabetical order', async () => {
+    const testSites: Site[] = [
+      { id: '1001', name: 'Site Zulu', address: 'Zulu Street' },
+      { id: '1002', name: 'Site Lima', address: 'Lima Street' },
+      { id: '1003', name: 'Site November', address: 'November Street' },
+      { id: '1004', name: 'Site Beta', address: 'Beta Street' },
+    ];
+    render(<SiteList sites={testSites} />);
+    const list = screen.getByRole('list');
```

Comment:
This is entirely a nitpick so feel free to ignore me, but I think we could make this block slightly more readable using the `within` method from react testing library: 

`    
const siteNames = within(screen.getByRole('list')).getAllByRole('listitem');

expect(siteNames.length).toBe(4);
expect(siteNames[0]?.textContent).toEqual('Site Beta');
expect(siteNames[1]?.textContent).toEqual('Site Lima');
expect(siteNames[2]?.textContent).toEqual('Site November');
expect(siteNames[3]?.textContent).toEqual('Site Zulu');
`

Reply:
In fact that first check could even be `expect(siteNames).toHaveLength(4);` ..... sorry, obviously not an important change but just thought of it!

Reply:
All done :)